src/cpl.rs

```diff
@@ -67,14 +71,30 @@ impl CslStringList {
         Ok(value)
     }
 
+    /// Determine the number of entries in the list.
+    pub fn len(&self) -> isize {
+        unsafe { CSLCount(self.as_ptr()) as isize }
```

Comment:
Nit:

```suggestion
        unsafe { CSLCount(self.as_ptr()) } as isize
```

Reply:
@lnicola Had to make this `(unsafe { CSLCount(self.as_ptr()) }) as isize` to parse.